MParT/MultiIndices/MultiIndex.h

```diff
@@ -50,6 +50,19 @@ class MultiIndex {
         }
     }
 
+    /**
+     * @brief Construct a new Multi Index object from a set of nonzero indices and values
+     * 
+     * @param nzIndsIn indices of the nonzero values
+     * @param nzValsIn values that are nonzero
+     * @param numNz number of nonzero values (i.e., length of nzIndsIn and nzValsIn)
+     * @param lengthIn dimension of the index (numNz==lengthIn iff all values are nonzero)
+     */
+    MultiIndex(unsigned int* nzIndsIn,
```

Comment:
I think this would be safer if it didn't use raw pointers.  Instead, could we use Kokkos views and use a subview when we call this constructor in `FixedMultiIndexSet::Unfix`?